packages/account/src/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,177 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper, Loading, DataTable } from '@deriv/components';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
+import LoadErrorMessage from 'Components/load-error-message';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history, is_dark_mode }) => (
+    <table
+        className={classNames('login-history-table', {
+            'theme-dark': is_dark_mode,
+        })}
+    >
+        <tbody>
+            <tr>
+                <Td title={fields.date} text={login_history.date} />
+            </tr>
+            <tr>
+                <Td title={fields.browser} text={login_history.browser} />
+                <Td className='action' title={fields.action} text={login_history.action} />
+            </tr>
+            <tr>
+                <Td title={fields.ip} text={login_history.ip} />
+                <Td title={fields.status} text={login_history.status} />
+            </tr>
+        </tbody>
+    </table>
+);
+
+const parseUA = (user_agent) => {
```

Comment:
Can we move this to `shared` as a helper? Since the mapping of browser names can be re-used for other purposes without the repetition of the mapping.